docs/source-pytorch/fabric/guide/checkpoint.rst

```diff
@@ -35,7 +35,7 @@ To save the state to the filesystem, pass it to the :meth:`~lightning_fabric.fab
 
     fabric.save("path/to/checkpoint.ckpt", state)
 
-This will unwrap your model and optimizer and convert their `state_dict` automaticall for you.
+This will unwrap your model and optimizer and convert their `state_dict` automatically for you.
```

Comment:
```suggestion
This will unwrap your model and optimizer and convert their `state_dict` automaticall for you.
```
Typo fix moved to #16480 so it doesn't require docs codeowners review for this PR